arm-apimanagement/2016-07-07/swagger/apimmanagement.json

```diff
@@ -6837,6 +7157,17 @@
       "maxLength": 256,
       "pattern": "^[^*#&+:<>?]+$",
 	    "x-ms-parameter-location": "method"
+    },
+    "BackendIdParameter": {
+      "name": "backendid",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "User identifier. Must be unique in the current API Management service instance.",
+      "minLength": 1,
+      "maxLength": 255,
+      "pattern": "^[^*#&+:<>?]+$",
+	    "x-ms-parameter-location": "method"
```

Comment:
#fixed

---
In reply to: [87418041](https://github.com/Azure/azure-rest-api-specs/pull/705#discussion_r87418041) [](ancestors = 87418041)